README.rst

```diff
@@ -1,6 +1,24 @@
-===============
- REANA cluster
-===============
+=============
+REANA Cluster
+=============
```

Comment:
The formatting was OK, see "Section title" in reST docs http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections